test/python/exchange_partner_identity.py

```diff
@@ -0,0 +1,55 @@
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives.asymmetric import ec, dh
+from cryptography.hazmat.primitives import serialization, hashes
+
+
+# A helper class to simulate an Exchange partner
+class ExchangePartnerIdentity:
```

Comment:
This class and `LedgerTestSigner` are kinda the same really. Maybe they could be merged into a higher-level class?

Reply:
Merged